build.sbt

```diff
@@ -21,6 +21,7 @@ val coreDependencies = Seq(
   "org.slf4j" % "log4j-over-slf4j" % "1.7.12",
   "org.scalatest" %% "scalatest" % "2.2.4" % "test",
   "com.typesafe.scala-logging" %% "scala-logging-slf4j" % "2.1.2",
+  "org.scalaj" %% "scalaj-collection" % "1.6",
```

Comment:
is this really necessary? there is [JavaConversions](http://www.scala-lang.org/files/archive/api/current/index.html#scala.collection.JavaConversions$) and [JavaConverters](http://www.scala-lang.org/files/archive/api/current/index.html#scala.collection.JavaConverters$) in the std library